frb_example/pure_dart/dart/lib/bridge_generated.dart

```diff
@@ -1103,6 +1116,10 @@ int api2wire_weekdays(Weekdays raw) {
 
 // Section: wire2api
 
+DateTime _wire2api_Chrono_Utc(dynamic raw) {
+  return DateTime.fromMicrosecondsSinceEpoch(raw, isUtc: true);
```

Comment:
What about trying this:

`return DateTime.fromMicrosecondsSinceEpoch(_wire2api_i64(raw), isUtc: true);`

Reply:
Oh dang I thought I tried it already (but obviously not)... it seems like it !